### PR TITLE
added password reset, change password, contact us, footer

### PR DESCRIPTION
**Pull request by emilysa**

### CODE REVIEW

**Comment by emilysa** [comment]
Not the prettiest it could be, but everything is functional!

File: genevieve/templates/account/password_reset_token.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+
+{% block main %}
+<form method="POST" action="{% url "account_password_reset_token" uidb36=uidb36 token=token %}">
+    {% csrf_token %}
```
Comment (madprime) [inline genevieve/templates/account/password_reset_token.html]: Python is 4-space indenting, but for HTML let's do 2-space indenting on HTML.

(HTML has a lot more nesting going on.)

File: genevieve/templates/pages/contact_us.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html' %}
+
+{% block content %}
+<div class="container">
+<h1>Contact us</h1>
+
+<p>For questions please contact us at
+<a href="mailto:genevieve-support@personalgenomes.org">genevieve-support@personalgenomes.org</a>.</p>
```
Comment (madprime) [inline genevieve/templates/pages/contact_us.html]: I'd format this differently to clarify indentation and make it more readable:

<p>
  For questions please contact us at
  <a href="...">email-address</a>.
</p>

File: genevieve/templates/pages/contact_us.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html' %}
+
+{% block content %}
+<div class="container">
```
Comment (madprime) [inline genevieve/templates/pages/contact_us.html]: turn into "main"

**Comment by madprime** [comment]
I this is ready to merge once the little formatting issues are addressed.

File: genevieve/templates/account/footer.html [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
```
Comment (madprime) [inline genevieve/templates/account/footer.html]: This file shouldn't exist.

**Pull request merged**